[2022-08-11 17:23:59,521] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2022-01-02T06:00:00+00:00 [queued]>
[2022-08-11 17:23:59,565] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2022-01-02T06:00:00+00:00 [queued]>
[2022-08-11 17:23:59,567] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 17:23:59,568] {taskinstance.py:1239} INFO - Starting attempt 6 of 6
[2022-08-11 17:23:59,572] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 17:23:59,616] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-01-02 06:00:00+00:00
[2022-08-11 17:23:59,622] {standard_task_runner.py:52} INFO - Started process 6344 to run task
[2022-08-11 17:23:59,629] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2022-01-02T06:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpx_dyrh91', '--error-file', '/tmp/tmpnoacdxcw']
[2022-08-11 17:23:59,632] {standard_task_runner.py:77} INFO - Job 55: Subtask ingest
[2022-08-11 17:23:59,770] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2022-01-02T06:00:00+00:00 [running]> on host cdc85d7e6248
[2022-08-11 17:23:59,930] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-02T06:00:00+00:00
[2022-08-11 17:23:59,935] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-11 17:23:59,938] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-08-11 17:23:59,950] {subprocess.py:85} INFO - Output:
[2022-08-11 17:24:00,033] {subprocess.py:89} INFO - ***.cfg
[2022-08-11 17:24:00,036] {subprocess.py:89} INFO - dags
[2022-08-11 17:24:00,038] {subprocess.py:89} INFO - logs
[2022-08-11 17:24:00,041] {subprocess.py:89} INFO - other.csv
[2022-08-11 17:24:00,044] {subprocess.py:89} INFO - output
[2022-08-11 17:24:00,048] {subprocess.py:89} INFO - output.gz
[2022-08-11 17:24:00,049] {subprocess.py:89} INFO - output_2021-01.csv
[2022-08-11 17:24:00,051] {subprocess.py:89} INFO - output_2021_01.csv.gz
[2022-08-11 17:24:00,052] {subprocess.py:89} INFO - output_2022-01.csv.gz
[2022-08-11 17:24:00,056] {subprocess.py:89} INFO - output_2022-05.csv.gz
[2022-08-11 17:24:00,059] {subprocess.py:89} INFO - plugins
[2022-08-11 17:24:00,060] {subprocess.py:89} INFO - requirements.txt
[2022-08-11 17:24:00,062] {subprocess.py:89} INFO - test
[2022-08-11 17:24:00,063] {subprocess.py:89} INFO - webserver_config.py
[2022-08-11 17:24:00,064] {subprocess.py:93} INFO - Command exited with return code 0
[2022-08-11 17:24:00,127] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20220102T060000, start_date=20220811T172359, end_date=20220811T172400
[2022-08-11 17:24:00,203] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-11 17:24:00,311] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
